_posts/en/newsletters/2019-11-27-newsletter.md

```diff
@@ -0,0 +1,301 @@
+  mailing list][lnd engineering] hosted by Google Groups was announced
+  for LND application developers, with an [initial post][osuntokun lnd
+  plans] by Olaoluwa
+  Osuntokun describing short-term goals for LND's next release.
+  Separately, the existing mailing lists for [Bitcoin-Dev][] and
+  [Lightning-Dev][] have recently had their hosting [transferred][togami
+  ml update] to Oregon State University's [Open Source Lab][osl] (OSL), a
+  well-respected organization that provides hosting to a variety of open
+  source projects.  Optech extends our thanks to everyone involved in
+  maintaining all of Bitcoin's many open communication channels, without
+  which this newsletter wouldn't exist.
+
+- **Schnorr/Taproot updates:** participants in the [taproot review
+  group][] have been continuing their review of the proposed soft fork
+  changes to Bitcoin, with many interesting questions being asked and
+  answered in the [logged][tbr log] ##taproot-bip-review IRC chatroom on
+  the Freenode network.  Additionally, some participants have been
+  writing their own implementations of parts of the BIPs, including for
+  the libbitcoin and bcoin full verification nodes.
+
+    Two informative blog posts were also posted this week related to the
+    security of multiparty schnorr signatures.  Blockstream engineer
+    Jonas Nick [describes][nick musig] the [MuSig][] multiparty
+    signature scheme designed to allow users of [bip-schnorr][] to
+    aggregate multiple public keys into a single pubkey that they can
+    later sign for using a single signature generated collaboratively
+    among them.  Nick describes the three steps of the MuSig signing
+    protocol---the exchange of nonce commitments, the exchange of
+    nonces, and the exchange of signatures (with the nonces and the
+    partial signatures then being aggregated to produce the final
+    signature).  To save time when speed is critical (e.g. in creating
+    LN channel commitment transactions), some people might want to
+    exchange nonce commitments and then nonces before they know what
+    transaction they want to commit to with their signature---but this
+    is unsafe due to Wagner's algorithm, as Nick briefly explains.  The
+    only information that can be safely shared before each participant
+    knows the transaction to sign is the nonce commitment.  (Not
+    mentioned in the blog post, but discussed on IRC, was that Pieter
+    Wuille and others have been investigating a construction based on a
+    Zero Knowledge Proof (ZKP) that could allow reduced interactivity.)
+    The blog post concludes with a suggestion that interested readers
+    review the MuSig implementation in [libsecp256k1-zkp][], which is
+    designed to help developers use the protocol safely.
+
+    Influenced by Jonas Nick's presentation on this topic at the Berlin
+    Lightning Conference, Adam Gibson wrote a separate [blog
```

Comment:
+1 for including this. I always find waxwing's posts very clear and informative. :+1: